src/com/unbounce/dogstatsd/core.clj

```diff
@@ -41,29 +41,29 @@
   (when-not (and client once?)
     (shutdown!)
     (alter-var-root #'client (constantly
-                              (com.timgroup.statsd.NonBlockingStatsDClient.
+                              (NonBlockingStatsDClient.
```

Comment:
This is covered by the import line in the module above, is it necessary to fully qualify this?

Reply:
Exactly, the class is imported into the current namespace, so I changed the code to refer to it directly instead of fully qualifying it. (Not sure I understand your question?)

This is just a small clean-up edit, unrelated to the new feature. Fine to revert it if you'd prefer the fully qualified name.

Reply:
ah okay, I gotcha. Sorry I was reading the PR backwards 😅